website/components/LandingPage/index.tsx

```diff
@@ -1,12 +1,15 @@
 import { faUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Button, Tooltip } from "antd";
-import React, { ReactElement, useEffect, useState } from "react";
+import React, { ReactElement, useEffect, useMemo, useState } from "react";
 
 import { landingPageContent } from "./content";
-import { DatasetEntry, ProjectEntry, ViewerArgs } from "../../types";
+import { AppDataProps, DatasetEntry, ProjectEntry } from "../../types";
 import styled from "styled-components";
 import { FlexColumnAlignCenter, FlexColumn, FlexRowAlignCenter, VisuallyHidden, FlexRow } from "./utils";
+import { useNavigate } from "react-router";
+import { getArgsFromParams } from "../../utils/url_utils";
+import { useSearchParams } from "react-router-dom";
```

Comment:
minor nit: imports could be sorted up

Reply:
I should add automatic import sorting...